scripts/test/test-utils.sh

```diff
@@ -1,37 +1,8 @@
-#!/bin/bash
```

Comment:
Not a big deal, but it is only used from `run-tests.sh` so it is not necessary to have split these functions.

Reply:
True, but I like the separation. It makes `run-tests.sh` very easy to read.